README.md

```diff
@@ -85,6 +85,9 @@ pip install -U pip setuptools wheel
 
 # install the package in editable mode
 pip install -e .[dev]
+# if you use zsh, pip install will fail. Use this instead:
+pip install -e ."[dev]"
```

Comment:
Thanks for adding this note :+1: 
I'm not user of zsh but I think it will be great if treat as full string
```
pip install -e ".[dev]"
```

Reply:
@digitronik okay, that works too. Will update it.